### PR TITLE
Rename test resource good_links.yaml to bad_links_missing_building.yaml

### PR DESCRIPTION
**Pull request by nkilmer**

### CODE REVIEW

**Comment by nkilmer** [comment]
Hmm... so it seems that another test failed because it was relying on good_links.yaml. I feel like something's wrong in the test design when one test uses the file as a success case and another uses it as a failure case. I'm also confused because the test in handler_test.py expects (correctly) the parser to raise a SyntaxError on this file, yet the test that's now failing in entity_instance_test.py expects the file to be parsed successfully...

**Comment by charbull** [comment]
@nkilmer tests are failing PTAL

File: tools/validators/instance_validator/tests/handler_test.py [hunk 1]
```diff
@@ -39,8 +39,8 @@ def testValidateOneBuildingExist(self):
 
   def testValidateOneBuildingExistFails(self):
     with self.assertRaises(SyntaxError):
-      # there is missing building type in the test file
-      input_file = path.join(_TESTCASE_PATH, 'GOOD', 'good_links.yaml')
+      input_file = path.join(
+        _TESTCASE_PATH, 'BAD', 'bad_links_missing_building.yaml')
```
Comment (charbull) [inline tools/validators/instance_validator/tests/handler_test.py:43]: just rename to missing_building.yaml?

**Review by charbull — APPROVED** [review]

**Comment by charbull** [comment]
Yes there is a test which depends on the good yaml
So you can just add a bad_missing_building.yaml file and leave the other one?

**Comment by charbull** [comment]
@nkilmer can you please take a look

**Comment by nkilmer** [comment]
> @nkilmer can you please take a look

Fixed.

**Review by charbull — APPROVED** [review]

**Pull request merged**